src/components/Card/index.js

```diff
@@ -10,44 +10,44 @@ const Card = ({ frontmatter, fields }) => {
     return (
         <CardWrapper fixed={!!frontmatter.abstract}>
             <div className="post-block">
-                <div className="post-thumb-block">
-                    <Link className="anchor" to={fields.slug}>
+                <Link className="anchor" to={fields.slug}>
+                    <div className="post-thumb-block">
                         <Image
                             {...frontmatter.thumbnail}
                             imgStyle={{ objectFit: "contain" }}
                             alt={frontmatter.title}
                         />
-                    </Link>
-                </div>
-                <div className="post-content-block">
-                    <h2 className="post-title">
-                        <Link className="anchor" to={fields.slug}>
-                            {frontmatter.title}
-                        </Link>
-                    </h2>
-                    <div className="post-meta-block">
-                        {frontmatter.author && <span>By: {frontmatter.author}</span>}
-                        {frontmatter.date && (
-                            <>
-                                <span className="divider">/</span>
-                                <span>{frontmatter.date}</span>
-                            </>
-                        )}
-                        {frontmatter.abstract && (
-                            <p className="post-entry">{frontmatter.abstract}</p>
-                        )}
                     </div>
-                    <div className="readmore-btn-wrapper">
-                        <Link className="readmore-btn" to={fields.slug}>
-                            see more <IoIosArrowRoundForward />
-                        </Link>
-                        {frontmatter.eurl && (
-                            <a className="readmore-btn" href={frontmatter.eurl}>
-                                <BiLinkExternal />
-                            </a>
-                        )}
+                    <div className="post-content-block">
+                        <h2 className="post-title">
+                            <Link className="anchor" to={fields.slug}>
```

Comment:
There is no need for a `Link` tag above these when it's present on the entire card. Do this for other places

Reply:
It was discussed to have the entire card clickable.

Reply:
So just put a link tag on the entire card instead of individual places